### ISSUE TITLE
Include Travis in the process

### ISSUE THREAD

**Issue by beatrupp** [mstarke/MacPass#634]
May I suggest that you setup the Github project for MacPass in such a way that a pull request needs a passing Travis build in order to being mergeable?

**Comment by mstarke** [comment]
Good idea. I'll look into it.

**Comment by mstarke** [comment]
Good point. The continuous branch is for automated builds so it might be useful to diverge there and add just testing for the other branches but I'm not sure this is a simple setup for travis. On the other hand, test coverage for MacPass is rather slim. KeePassKit on the other hand has much better test coverage.

**Comment by mstarke** [comment]
This the recent changes GitHub Actions it the way to go so Travis is out for now.
